docs/user-guide/a1-06-spirv-target-specifics.md

```diff
@@ -224,23 +224,48 @@ When a slang module uses a pointer, the resulting SPIRV will be using the SpvAdd
 Matrix type translation
 -----------------------
 
-TODO: Quickly summarize what DXC does as described in [HLSL to SPIR-V Feature Mapping Manual](https://github.com/Microsoft/DirectXShaderCompiler/blob/main/docs/SPIR-V.rst#appendix-a-matrix-representation) and say Slang follows the same behavior.
+As an example, a Slang shader can iterate each element of a `float3x4` matrix as following,
```

Comment:
Start with the conclusion:

A m-row-by-n-column matrix in Slang, represented as `floatmxn` or `matrix<T, m, n>`, is represented as `OpTypeMatrix (OpTypeVector(T, n), m)` in SPIRV. Note that in SPIR-V terminology, this type is referred to a m-column-by-n-row matrix.

The swap of row and column terminology may seem to be confusing at first, but this is the most sensical translation that avoids extra operations that can potentially have negative performance impact. For example, consider the following Slang code: